test/freetype_test.py

```diff
@@ -669,7 +669,9 @@ def test_freetype_Font_render_to(self):
         rrect = font.render_to(surf, (32, 32), "FoobarBaz", color, None, size=24)
         self.assertIsInstance(rrect, pygame.Rect)
         self.assertEqual(rrect.top, rrect.height)
-        ## self.assertEqual(rrect.left, something or other)
+        ##test that render_to is position
+        self.assertEqual(rrect.left, 32)
+        self.assertEqual(rrect.right, 32)
```

Comment:
This seems wrong, the returned rect (according to the docs) is supposed to have the size as well as the position of the rendered text - so the right hand side of the rectangle should definitely be further to the right right than the left hand side with the text 'FoobarBaz' at font size 24.

**Example picture:**

![image](https://user-images.githubusercontent.com/13382426/85527109-7b1e6200-b602-11ea-8f15-cf164d37c7ba.png)

It won't come out to exactly 156px on the right hand side of the rect as that's m,e just eyeballing it in a paint program, but it should definitely have at least _some_ width.